static/js/services/db-sync.js

```diff
@@ -97,24 +95,40 @@ var _ = require('underscore'),
           });
       };
 
-      return function() {
+      return function(replicateDoneListener) {
```

Comment:
The naming convention is "callback". Listener implies EventListener (to me) that is a different pattern entirely.